src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/model/Status.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.riskprofiler.model;
+
+public enum Status {
+    NEW, ADDED_TODO, IGNORED
```

Comment:
Are all these status used?  Can only see NEW

Reply:
Not yet, theyre just a guess at what will be needed